resources/views/header.blade.php

```diff
@@ -0,0 +1,9 @@
+<nav class="bg-gray-800 py-2 md:py-4">
+    <div class="hidden md:flex flex-col md:flex-row md:ml-auto mt-3 md:mt-0" id="navbar-collapse">
+        <a href="/" class="p-2 lg:px-4 md:mx-2 text-white rounded bg-indigo-600">Home</a>
+        <a href="#" class="p-2 lg:px-4 md:mx-2 text-white rounded hover:bg-gray-200 hover:text-gray-700 transition-colors duration-300">Cities</a>
+        <a href="/airline" class="p-2 lg:px-4 md:mx-2 text-white rounded hover:bg-gray-200 hover:text-gray-700 transition-colors duration-300">Airlines</a>
+        <a href="#" class="p-2 lg:px-4 md:mx-2 text-white rounded hover:bg-gray-200 hover:text-gray-700 transition-colors duration-300">Flights</a>
```

Comment:
```suggestion
        <a href="/airline" class="p-2 lg:px-4 md:mx-2 text-white rounded hover:bg-gray-200 hover:text-gray-700 transition-colors duration-300">Airlines</a>
```
Given the fact that the other 2 modules aren't developed yet, skip them from the navigation bar.